### PR TITLE
[Policy] Update Personal Ownership checkbox description

### PR DESCRIPTION
**Pull request by vincentsalucci**
## Code Changes
- **policy-edit.component.html**: Use new `getCheckboxDesc` to properly retrieve the enable checkbox text based on the policy type being edited
- **policy-edit.component.ts**: Added `getCheckboxDesc` that will return a different string based on the Policy Type being edited
- **message.json**: Added new string

## Screenshot
0-web-enabled-checkbox

### CODE REVIEW

File: src/app/organizations/manage/policy-edit.component.ts [hunk 1]
```diff
@@ -172,6 +172,11 @@ export class PolicyEditComponent implements OnInit {
         }
     }
 
+    getCheckboxDesc(): string {
+        return this.type === PolicyType.PersonalOwnership ? this.i18nService.t('personalOwnershipCheckboxDesc') :
+            this.i18nService.t('enabled');
+    }
+
```
Comment (addisonbeck) [inline src/app/organizations/manage/policy-edit.component.ts:179]: This is just syntax sugar, so feel free to disregard, but this reads better as a getter to me.

```suggestion
    get checkboxDesc(): string {
        return this.type === PolicyType.PersonalOwnership ? this.i18nService.t('personalOwnershipCheckboxDesc') :
            this.i18nService.t('enabled');
    }

```

File: src/app/organizations/manage/policy-edit.component.html [hunk 2]
```diff
@@ -36,7 +36,7 @@ <h2 class="modal-title" id="policiesEditTitle">{{'editPolicy' | i18n}} - {{name}
                     <div class="form-check">
                         <input class="form-check-input" type="checkbox" id="enabled" [(ngModel)]="enabled"
                             name="Enabled">
-                        <label class="form-check-label" for="enabled">{{'enabled' | i18n}}</label>
+                        <label class="form-check-label" for="enabled">{{getCheckboxDesc()}}</label>
```
Comment (addisonbeck) [inline src/app/organizations/manage/policy-edit.component.html:39]: Only needed if the method in the component ends up getting changed too.

```suggestion
                        <label class="form-check-label" for="enabled">{{checkboxDesc}}</label>
```

**Review by addisonbeck — CHANGES_REQUESTED** [review]

File: src/app/organizations/manage/policy-edit.component.html (hunk 2 above)
Comment (vincentsalucci) [reply src/app/organizations/manage/policy-edit.component.html]: Updated!

File: src/app/organizations/manage/policy-edit.component.ts (hunk 1 above)
Comment (vincentsalucci) [reply src/app/organizations/manage/policy-edit.component.ts]: Updated!

File: src/locales/en/messages.json [hunk 3]
```diff
@@ -3586,8 +3586,13 @@
   "custom": {
     "message": "Custom"
   },
+<<<<<<< HEAD
```
Comment (Hinton) [inline src/locales/en/messages.json:3589]: Merge conflict :(

**Review by Hinton — CHANGES_REQUESTED** [review]

**Review by Hinton — APPROVED** [review]

**Review by addisonbeck — APPROVED** [review]

File: src/locales/en/messages.json (hunk 3 above)
Comment (vincentsalucci) [reply src/locales/en/messages.json]: Sorry - fixed.

**Pull request merged**